src/index.js

```diff
@@ -6,4 +6,9 @@ export default [
   'The leak is minor, we’ll just add more RAM later.',
   'Looks like it’s all working fine',
   'We’ll fix it later',
+  'If it works it works.',
+  'Let’s build our own framework for it.',
```

Comment:
Hm, building your own plugin is mostly a bad idea, especially if you are a small startup - however lots of frameworks begun like this, such as React. I have mixed thoughts about it.

Reply:
Its ironic coming from me when i mostly build anything i can myself but i was thinking more about overengineering. Inspired by https://www.youtube.com/watch?v=Wm2h0cbvsw8.